azurecontainerapps.ts

```diff
@@ -537,19 +537,30 @@ export class azurecontainerapps {
                 this.commandLineArgs.push(`--env-vars ${environmentVariables}`);
             }
         }
+
+        if (!this.imageToDeploy.startsWith(this.defaultRegistryServer)) {
+            this.commandLineArgs.push(`-i ${this.imageToDeploy}`);
```

Comment:
Nit: should this be in an `else` block of the other condition below? Or at least an `else if`, to make sure that both arguments can never be set together?

Reply:
That sounds possible? but need to verify it does not break existing scenarios. It may probably be a better solution set the value for `imageToDeplow` only for scenarios handled by the GitHub action? 
I can get another PR out for this change. @daniv-msft Is this something we want to be checked by today or can get another for this tomorrow?